cmd/codegen/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"io/fs"
+	"io/ioutil"
+	"path/filepath"
+)
+
+func main() {
+
+	codegen := codeGen{}
+	v1Config := codegen.createV1Config()
+	err := ioutil.WriteFile(filepath.Join(".", "pkg", "integrations", "v1", "config.go"), []byte(v1Config), fs.ModePerm)
```

Comment:
Tiny nit: use 0664 here, `fs.ModePerm` is 0777 and gives the generated files the execute bit :)